Examples cleanup

reinierdevalk opened the pull request:
Updated the four tablature examples and three of the four guitar examples according to what we discussed last December. Also cleaned the documentation here and there, but nothing substantial.

reinierdevalk committed:
cleanup Da Crema

reinierdevalk committed:
cleanup milano, some renaming

reinierdevalk committed:
cleanup four lute examples; some renaming and deduplication; added gitignore

reinierdevalk committed:
cleanup four lute examples; some renaming and deduplication; added gitignore

reinierdevalk committed:
some small corrections

reinierdevalk committed:
another correction

reinierdevalk committed:
small corrections to lute tabs; completed three guitar tabs

DILewis commented:
This is looking great to me. Some thoughts:
 * I would prefer we resolve the things you comment before the pull happens - XML comments aren't available as part of MEI semantics, so if we want to say something, we should either do it in MEI or delete it
 * I'd love for the directory structure to make sense – either everything in examples directly or maybe subdirectories for lute and guitar. Certainly not historical filing type ones like there are now
 * Metadata for the guitar examples, even if minimal, would be helpful
 * Where comments are observations (da crema, hook on the flag, which looks to me like an image processing artefact), they should probably be expressed within MEI. I can't think of how that might be achieved though.
 * In the da Milano, the sic should probably be part of a `<choice><sic /><corr /></choice>` structure, which would be cool to have anyway.
 * In the Satriani (Ice), maybe leave out the slur from before the piece starts. Having a non-existent ID could be problematic.
 * Not of this Earth: "Fdbk." should probably be a `<dir>` or something similar

reinierdevalk commented:
@DILewis I don't understand this:

> I'd love for the directory structure to make sense – either everything in examples directly or maybe subdirectories for lute and guitar. Certainly not historical filing type ones like there are now.

Can you be a bit more specific? There is a lot of stuff in the `examples/` directory of which I don't know why it's there. Maybe this could go into a `misc` directory (I did this for now)?

reinierdevalk committed:
addressed issues brought up on GitHub

reinierdevalk commented:
@DILewis @TimCrawford I addressed all of David's comments above and updated the branch (and the Google doc). 

I made the following changes:
- Guitar pieces: added artist name and song name in `<workDesc>`. 
- Lute pieces: replaced all `<sic>`s with `<choice>` + `<sic>`, `<corr>`.
- Both: removed all XML comments and resolved the issue if possible, or added as a question or remark to the Google doc.
- Reorganised the `examples/` directory structure.

For remaining remarks and questions about details that need to be answered before I can merge, please see the Google doc (and please reply there)!

DILewis commented:
@reinierdevalk @TimCrawford This is a huge improvement. I've tried to be helpful about some of the issues @reinierdevalk raises in the Google Doc. I'm comfortable with some of them being agenda points for the SIG rather than blockers to this pull request. Some of them could become issues on the tracker.

lpugin commented:
This is excellent. One remark: we need to remove all the `@xml:id=''` since they are useless.

reinierdevalk commented:
@lpugin Useless but harmless, right? I left them in to be able to complete them at a later point. Would you rather have them removed completely if they are empty?

lpugin commented:
> Useless but harmless, right? I left them in to be able to complete them at a later point. Would you rather have them removed completely if they are empty?

Not quite, it makes validation fail, so I would remove them

reinierdevalk commented:
@lpugin Ah, I didn't know that. I will remove them.

reinierdevalk committed:
addressed latest issues as discussed on GitHub and Google doc

reinierdevalk commented:
@DILewis @lpugin @TimCrawford I pushed the following changes:

* Removed all empty `@xml:id`s
* Addressed all new Google doc replies
* Rephrased all remaining questions on the Google doc as discussion points for the SIG. 

Actually, maybe that list of issues should go to the GitHub as well, so we have everything in one place? After we agree on that last point, I think we're ready to merge.

reinierdevalk committed:
small correction

reinierdevalk committed:
added tab+CMN example

reinierdevalk committed:
small correction

reinierdevalk committed:
removed incorrect key and meter sigs from CMN example
